examples/confidential_computing/README.md

```diff
@@ -0,0 +1,28 @@
+# confidential computing vm
+
+This is an example of a vm creation with confidential computing, encrypted disk
+using a Cloud HSM key and a custom service account with cloud-platform scope.
```

Comment:
Indicate a multiregion HSM key so they don't have to lookup default in variables file.